symbology/read-symbols-from-mobile-style-file/src/main/java/com/esri/samples/read_symbols_from_mobile_style_file/ReadSymbolsFromMobileStyleFileController.java

```diff
@@ -104,7 +102,7 @@ public void initialize() {
    */
   private void loadSymbolsFromStyleFile() {
     // create a SymbolStyle with the .stylx file
-    File stylxFile = new File(System.getProperty("data.dir"), "./samples-data/stylx/emoji-mobile.stylx");
+    var stylxFile = new File(System.getProperty("data.dir"), "./samples-data/stylx/emoji-mobile.stylx");
```

Comment:
As commented above try using `resolve` (https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/nio/file/Path.html#resolve(java.lang.String)) as a nicer way to make paths and handle separators.

Reply:
My understanding is we're using the two-argument `File(String, String)` constructor as this will behave like `File(String)` if the first argument is null (which it will be unless the user has specified a different datadir themselves). I don't think `Path` resolves null so cleanly but could be wrong.

In the cases where we have file separators in the second argument I've been using `Path.of("path", "to", "the", "file").toString()` instead of `resolve`, which IMO is neater than chaining `resolve` calls. If there's a good reason to prefer `resolve` I'll switch to that.

Reply:
Handling of null is easily fixed with `System.getProperty("data.dir", "")`.